pkgs/by-name/ol/ollama/package.nix

```diff
@@ -199,20 +200,24 @@ goBuild {
     "-X=github.com/ollama/ollama/server.mode=release"
   ];
 
-  passthru.tests =
-    {
-      inherit ollama;
-      version = testers.testVersion {
-        inherit version;
-        package = ollama;
+  passthru = {
+    tests =
+      {
+        inherit ollama;
+        version = testers.testVersion {
+          inherit version;
+          package = ollama;
+        };
+      }
+      // lib.optionalAttrs stdenv.isLinux {
+        inherit ollama-rocm ollama-cuda;
+        service = nixosTests.ollama;
+        service-cuda = nixosTests.ollama-cuda;
+        service-rocm = nixosTests.ollama-rocm;
       };
-    }
-    // lib.optionalAttrs stdenv.isLinux {
-      inherit ollama-rocm ollama-cuda;
-      service = nixosTests.ollama;
-      service-cuda = nixosTests.ollama-cuda;
-      service-rocm = nixosTests.ollama-rocm;
-    };
+
+    updateScript = nix-update-script { };
```

Comment:
I'm not familiar with this; is there any documentation on how updateScript works?

Reply:
https://nixos.org/manual/nixpkgs/stable/#var-passthru-updateScript

This should allow the bot to in theory open new pull requests, or you can use it locally with

```
nix-shell maintainers/scripts/update.nix --argstr package ollama --argstr commit true
```

Reply:
That's awesome, thanks for sharing the link to the docs, I was also curious about this one 🍻